.github/workflows/ci.yml

```
@@ -2407,7 +2410,7 @@ jobs:
           echo "✅ Homepage smoke passed"
 
   deploy:
-    needs: [ci-fast, ci-build, ci-unit-tests, ci-e2e-tests, ci-smoke-required] # Required smoke must pass before deploy; homepage smoke is informational
+    needs: [ci-fast, ci-build, ci-unit-tests, ci-e2e-tests] # Smoke tests run in parallel but don't block deploy; canary health gate verifies post-deploy
```

YAMLlint (1.38.0)

[warning] too few spaces before comment: expected 2

(comments)